workos/types/user_management/user.py

```diff
@@ -6,6 +6,9 @@
 
 class User(WorkOSModel):
     """Representation of a WorkOS User."""
+    options: Optional[str] = field(default=None)
+    userId: Optional[str] = field(default=None)
+    userId: Optional[str] = field(default=None)
```

Comment:
did we mean to leave a duplicate field?

Reply:
Ack, I meant to close this. It's an example of an automated PR generated from another as a AI hacking project from the onsite.